src/sst/elements/vanadis/util/vfpreghandler.h

```diff
@@ -46,6 +46,7 @@ fractureToRegisters(VanadisRegisterFile* regFile, uint16_t r_left, uint16_t r_ri
     std::memcpy((char*)&left_val, value_ptr, sizeof(left_val));
 
     uint32_t right_val = 0;
+    if(sizeof(T) == 8)
```

Comment:
Formatting

Reply:
I did not use `clang-format` because the changes would be too large, and even if I indented the code in the `if` statement, it would add whitespace changes which distract from the important diffs emphasized by this PR. The diffs were minimized to fix warnings or bugs, but not to reformat.